Code/wwlib/cpudetect.cpp

```diff
@@ -1022,7 +1038,7 @@ void CPUDetectClass::Init_Processor_Log()
 	}
 
 	if (CPUDetectClass::Get_L1_Instruction_Trace_Cache_Size()) {
-		SYSLOG(("L1 Instruction Trace Cache: %d way set associative, %dk �OPs\r\n",
+		SYSLOG(("L1 Instruction Trace Cache: %d way set associative, %dk �OPs\r\n",
```

Comment:
UTF-8 issue again?

Reply:
Let's make the code ascii :)
```suggestion
		SYSLOG(("L1 Instruction Trace Cache: %d way set associative, %dk uOPs\r\n",
```